sdk/test-utils/testing-recorder-new/test/core-v1-test.spec.ts

```diff
@@ -45,7 +47,11 @@ describe("Core V1 tests", () => {
   });
 
   it("storage-queue create queue", async function() {
-    const client = new QueueServiceClient(env.STORAGE_SAS_URL, undefined, { httpClient: recorder });
+    const client = new QueueServiceClient(
+      getEnvironmentVariable("STORAGE_SAS_URL"),
```

Comment:
Maybe this function should be called "assert" instead of "get" to make it clear that it is not falsifiable and will throw if the variable isn't defined.

Reply:
updated